docs/configuration/git-providers/create-github-app.md

```diff
@@ -1,28 +1,44 @@
-# Create and configure a new GitHub App for GitHub Enterprise
-
-## Creating a new GitHub App
-
-Navigate to your GitHub installation Settings > Developer Settings > GitHub Apps > New GitHub App or https://github.example.com/settings/apps/new, where github.example.com is your GitHub Enterprise hostname.
-
-
-| Field                                   | Value                                                 |
-| --------------------------------------- | ----------------------------------------------------- |
-| GitHub App name                         | Codacy                                                |
-| Homepage URL                            | https://codacy.example.com                            |
-| User authorization callback URL         | https://codacy.example.com                            |
-| Webhook URL                             | https://codacy.example.com/2.0/events/gh/organization |
-| Repository permissions                  |
-| Administration                          | Read & Write                                          |
-| Checks                                  | Read & Write                                          |
-| Issues                                  | Read & Write                                          |
-| Metadata                                | Read Only                                             |
-| Pull requests                           | Read & Write                                          |
-| Webhooks                                | Read & Write                                          |
-| Commit statuses                         | Read & Write                                          |
-| Organization permissions                |
-| Members                                 | Read Only                                             |
-| Webhooks                                | Read & Write                                          |
-| User permissions                        |
-| Email addresses                         | Read Only                                             |
-| Git SSH keys                            | Read & Write                                          |
-| Where can this GitHub App be installed? | Any account                                           |
+# Creating a GitHub App
+
+You must create and correctly set up a [GitHub App](https://developer.github.com/apps/about-apps/) to allow Codacy to integrate with GitHub.
+
+To create the GitHub App:
+
+1.  **If you're using GitHub Cloud**, open <https://github.com/settings/apps/new>.
+
+    **If you're using GitHub Enterprise**, open `https://<github hostname>/settings/apps/new`, where `<github hostname>` is the hostname of your own GitHub Enterprise instance.
```

Comment:
I think it's more direct and simply faster to just provide the direct URL to the page where the users can create their GitHub Apps.